docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1b.md

```diff
@@ -82,7 +82,7 @@ Creating cluster role "kubestellar-syncer-florin-32uaph9l" to give service accou
 
 Creating or updating cluster role binding "kubestellar-syncer-florin-32uaph9l" to bind service account "kubestellar-syncer-florin-32uaph9l" to cluster role "kubestellar-syncer-florin-32uaph9l".
 
-Wrote WEC manifest to florin-syncer.yaml for namespace "kubestellar-syncer-florin-32uaph9l". Use
+Wrote workload execution cluster manifest to florin-syncer.yaml for namespace "kubestellar-syncer-florin-32uaph9l". Use
```

Comment:
@nirrozenbaum would you mind putting "(WEC)" next to each place where you expanded into 'workload execution cluster'?

Reply:
sure. I'll push it very soon